scripts.js

```diff
@@ -1 +1,132 @@
-console.log('hello world')
\ No newline at end of file
+// Task 1 - Add a button and an input to your HTML.
+// Every time the user clicks the button, change the background color of the button to the color typed by the user.
+
+function changeButtonColor() {
+    const colorInput = document.querySelector("#userColorInput").value;
```

Comment:
```suggestion
function changeButtonColor(event) {
    const colorInput = document.querySelector("#userColorInput").value;
```
use event to obtain target element

Reply:
`event.target`